pkg/config/centralconfig.go

```diff
@@ -488,6 +488,17 @@ func (c *CentralConfiguration) validateConfig() {
 	}
 }
 
+func (c *CentralConfiguration) validateURL(urlString, configPath string, isURLRequired bool) {
```

Comment:
I guess I don't understand the boolean?  when is the URL not required?  If its not, then why even call validateURL?

Reply:
A few urls, like those for Proxys, are not required in the config. But, if they are non-blank, they should be validated.